includes/Customer.php

```diff
@@ -10,24 +10,87 @@
  */
 class Customer {
 
+    /**
+     * Normalized, weekly-cached.
+     *
+     * @var string
+     */
+    private static $transient = 'bh_cdata';
```

Comment:
Probably should just use `const` here (and below)?

Reply:
Pushed a commit to address @wpscholar